test/.default_config

```diff
@@ -1,3 +1,3 @@
-tests.build-image-version=0.10.0-rc1
-tests.services-image-version=0.10.0-rc1
-tests.examples-ref=master
```

Comment:
the examples reference should stay pointing to master

Reply:
yep, the bump-version is guessing to have 0.11.x branch already. will update the bump-version script

Reply:
https://github.com/kiegroup/kogito-cloud-operator/pull/321